scripts/zkai_native_seq32_attention_mlp_median_timing_gate.py

```
@@ -0,0 +1,499 @@
+    except Exception as err:  # noqa: BLE001 - normalize imported validator failures.
+        raise Seq32TimingGateError(f"statement-only gate validation failed: {err}", layer="source_statement_gate") from err
+    expect_equal(payload.get("schema"), STATEMENT_ONLY_SCHEMA, "statement-only schema", layer="source_statement_gate")
+    expect_equal(payload.get("decision"), STATEMENT_ONLY_DECISION, "statement-only decision", layer="source_statement_gate")
+    expect_equal(payload.get("result"), STATEMENT_ONLY_RESULT, "statement-only result", layer="source_statement_gate")
+    return copy.deepcopy(payload)
+
+
+def statement_only_best_row() -> dict[str, Any]:
+    rows = require_list(statement_only_payload().get("profile_rows"), "statement-only profile rows", layer="source_statement_gate")
+    for row in rows:
+        row = require_object(row, "statement-only profile row", layer="source_statement_gate")
```

Ruff (0.15.13)

[warning] `for` loop variable `row` overwritten by assignment target

(PLW2901)